play-java-ebean-example/build.sbt

```diff
@@ -8,11 +8,16 @@ lazy val root = (project in file(".")).enablePlugins(PlayJava, PlayEbean)
 
 libraryDependencies += guice
 libraryDependencies += jdbc
-libraryDependencies += "com.h2database" % "h2" % "1.4.197"
+libraryDependencies += "com.h2database" % "h2" % "1.4.199"
 
 libraryDependencies += "org.awaitility" % "awaitility" % "2.0.0" % Test
 libraryDependencies += "org.assertj" % "assertj-core" % "3.6.2" % Test
 libraryDependencies += "org.mockito" % "mockito-core" % "2.1.0" % Test
 testOptions in Test += Tests.Argument(TestFrameworks.JUnit, "-a", "-v")
 
 javacOptions ++= Seq("-Xlint:unchecked", "-Xlint:deprecation", "-Werror")
+
+// To provide an implementation of JAXB-API, which is required by Ebean.
```

Comment:
Should this be reported upstream in ebean?

Reply:
Not sure since this only makes sense to Java 11 (although I don't think it will hurt to have it for Java 8 too). WDYT?

Reply:
Well, it's a question for the Ebean maintainers, but I think there's an argument that it should work out-the-box on the latest, stable, LTS version of Java.